### PR TITLE
Cover image

### PR DESCRIPTION
**Pull request by warpfox**
Adds the ability to give a draft/adventure a cover image.bitmoji

### CODE REVIEW

File: client/src/editor/components/Draft/Draft.js [hunk 1]
```diff
@@ -124,13 +136,49 @@ const Draft = ({
...
           <button onClick={handlePublishClick}>PUBLISH ADVENTURE</button>
         </div>
+        <>
+          {draft.coverImage ? (
+            <div>
+              <div>{`Your current cover image: ${draft.coverImage}`}</div>
+              <div>
+                <img
+                  style={{
+                    position: 'relative',
```
Comment (VirtuaBoza) [inline client/src/editor/components/Draft/Draft.js]: We should be consistent and use css modules for all styles

File: client/src/editor/components/Drafts/Drafts.js [hunk 2]
```diff
@@ -5,6 +5,7 @@ import * as routes from '../../constants/routes';
 import { draftsSelector } from '../../store/selectors';
 
 const Drafts = ({ drafts }) => {
+  console.log(drafts);
```
Comment (VirtuaBoza) [inline client/src/editor/components/Drafts/Drafts.js]: delete

File: client/src/reader/components/AdventureListItem/AdventureListItem.js [hunk 3]
```diff
@@ -22,6 +22,21 @@ const AdventureListItem = ({
 
   return (
     <li className={styles.container}>
+      <div>
+        <img
+          src={adventure.coverImage}
+          alt={adventure.title}
+          style={{
```
Comment (VirtuaBoza) [inline client/src/reader/components/AdventureListItem/AdventureListItem.js]: See above re: css modules

File: client/src/editor/components/Draft/Draft.js (hunk 1 above)
Comment (warpfox) [reply client/src/editor/components/Draft/Draft.js]: fixed

File: client/src/editor/components/Drafts/Drafts.js (hunk 2 above)
Comment (warpfox) [reply client/src/editor/components/Drafts/Drafts.js]: removed

File: client/src/reader/components/AdventureListItem/AdventureListItem.js (hunk 3 above)
Comment (warpfox) [reply client/src/reader/components/AdventureListItem/AdventureListItem.js]: fixed

**Pull request merged**